mise.toml

```diff
@@ -15,7 +15,7 @@ lockfile = false
 disable_backends = ["asdf"]
 
 [tools]
-buf = "1.55.1"
+"ubi:bufbuild/buf" = "1.55.1"
```

Comment:
Instead of explicitly setting the backend inline it might be nicer to define an alias:

```toml
[alias]
buf = "ubi:bufbuild/buf"
```

This worked for me testing on a local Windows machine and removes the need to modify the workflow files to call `mise tool ubi:bufbuild/buf` instead of `mise tool buf`